checks/compound/formatters/helper.ts

```diff
@@ -46,3 +47,16 @@ export function toYears(seconds: number, secondsPerYear = 31536000): number {
 export function calculateDifferenceOfDecimals(num1: number, num2: number): number {
   return defactor(factor(num1) - factor(num2))
 }
+
+export async function getPlatform(chain: CometChains) {
+  switch (chain) {
+    case CometChains.mainnet:
+      return 'etherscan.io'
+    case CometChains.polygon:
+      return 'polygonscan.com'
+    case CometChains.arbitrum:
+      return 'arbiscan.io'
+    case CometChains.base:
+      return ''
```

Comment:
Why do we have this as empty?

Reply:
I havent encountered any proposal relevant to this. just kept it as a placeholder